management/src/main/java/nearsoft/skytouch/management/service/ProductService.java

```diff
@@ -0,0 +1,24 @@
+package nearsoft.skytouch.management.service;
+
+import nearsoft.skytouch.common.model.Product;
+
+import java.util.List;
+
+/**
+ * Interface to define the methods that should be in the implementation to interact with the
+ * products it can be the way in which the products are retrieved or stored
+ */
+public interface ProductService {
+    /**
+     * Returns all the products
+     *
+     * @return a list with all the products
+     */
+    List<Product> retrieveProducts();
+
+    /***
+     * Service to store the products
+     * @param product the product to be stored
+     */
+    void storeProduct(Product product);
```

Comment:
I use to return the inserted Product. The instance returned should contain the id of the row and you can use that value for future updates.

Reply:
ok i agree maybe i will use the object later, thanks